js/src/components/google-mc-account-card/google-mc-account-card.js

```diff
@@ -3,19 +3,26 @@
  */
 import SpinnerCard from '.~/components/spinner-card';
 import useGoogleMCAccount from '.~/hooks/useGoogleMCAccount';
+import useGoogleAdsAccount from '.~/hooks/useGoogleAdsAccount';
 import ConnectedGoogleMCAccountCard from './connected-google-mc-account-card';
 import DisabledCard from './disabled-card';
 import NonConnected from './non-connected';
+import { GOOGLE_ADS_ACCOUNT_STATUS } from '.~/constants';
 
 const GoogleMCAccountCard = () => {
+	const { googleAdsAccount } = useGoogleAdsAccount();
 	const { hasFinishedResolution, isPreconditionReady, googleMCAccount } =
 		useGoogleMCAccount();
 
 	if ( ! hasFinishedResolution ) {
 		return <SpinnerCard />;
 	}
 
-	if ( ! isPreconditionReady ) {
+	if (
+		! isPreconditionReady ||
+		! googleAdsAccount ||
+		googleAdsAccount.status === GOOGLE_ADS_ACCOUNT_STATUS.DISCONNECTED
+	) {
```

Comment:
We want to allow for these connections to be made in either order, so these changes to this file need to be remove.